staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/dispatcher.go

```diff
@@ -169,8 +169,7 @@ func (a *mutatingDispatcher) Dispatch(ctx context.Context, attr admission.Attrib
 			switch err := err.(type) {
 			case *webhookutil.ErrCallingWebhook:
 				if ctx.Err() == context.Canceled {
-					klog.Warningf("Context Canceled when calling webhook %v", hook.Name)
-					return err
+					break
```

Comment:
I don't think we want to break here either, we probably want to flow through to the end

Reply:
I have added break statement here, so the metric counter in next steps wont pick the canceled context.